.test/meta-queue/fake-data.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+dir="$(dirname "$BASH_SOURCE")"
+
+tmpJson="$(mktemp)"
+trap 'rm $tmpJson' EXIT
+
+jq -n -c '
+	# input arch and test values
+	# output unique buildId string
+	def buildId($arch):
+		[$arch, .[] | tostring] | join("-")
+	;
+	[
+		# add new test cases here
+		# each item will be used for each architecture generated
+		# [ ".build.resloved", "count", "skips" ]
+		[ null, 1, 0 ], # buildable, tried once
+		[ null, 23, 0 ], # buildable, tried many but less than skip threshold
+		[ null, 24, 0 ], # buildable, tried many, just on skip threshold
+		[ null, 25, 23 ], # buildable, final skip
+		[ null, 25, 24 ], # buildable, no longer skipped
+		[ {}, 3, 0 ], # build "complete" (not queued or skipped)
+		empty # trailing comma
+	] as $data
+	| [ "amd64", "arm32v7", "windows-amd64" ]
+	| to_entries
```

Comment:
Do you still need this?  It seems like both of the following assign `.key` to `$archindex` but then never use it.